main.go

```diff
@@ -41,6 +42,9 @@ type DownloadStatus struct {
 	// some errors are ignored the download is retried instead of propagating
 	// the error).
 	Error error
+
+	// The size of the download in bytes
+	DowloadedFileSize uint64
```

Comment:
This variable already exists. It is declared at line 39.

Reply:
Actually, I got confused as we spoke about the DownloadedFileSize, the one available at main.go:39 is called DownloadedFileBytes (which technically is the same). Thanks for the reminder!

Reply:
Actually @danielfireman, isn't this variable a method to check for the total file size?

Reply:
> isn't this variable a method to check for the total file size?
 No, it's a field os the `struct` to hold the total file size.

Reply:
Awesome, tks @cuducos! Changing right now!